docker/start_server.sh

```diff
@@ -8,6 +8,8 @@ function symlink_client_files() {
     ln -s /client/client/res/names/ /app/res/names
     ln -s /client/client/res/CDServer.sqlite /app/res/CDServer.sqlite
     ln -s /client/client/locale/locale.xml /app/locale/locale.xml
+    # need to create this file so the server knows the client is unpacked (see `dCommon/dClient/AssetManager.cpp`)
+    touch /app/res/cdclient.fdb
```

Comment:
Does cdclient.fdb get deleted after the conversion process?  Not sure why this is needed if the file is needed by the client so some clarification here would be great

Reply:
It's still there, but it's just not linked to the location the server will run from. I tried linking it but it messed up and tried importing the db a second time. Didn't bother diving into why so I just created a basic file since the server just checks for existence.